flake.nix

```
@@ -12,6 +12,11 @@
     cardano-js-sdk.url = "github:input-output-hk/cardano-js-sdk/@cardano-sdk/cardano-services@0.35.10";
     cardano-js-sdk.flake = false; # we patch it & to prevent lockfile explosion
 
+    blockfrost-platform = {
+      url = "github:blockfrost/blockfrost-platform/0.0.2";
```

```suggestion
      url = "github:blockfrost/blockfrost-platform?ref=feat/addresses_utxos"
```

Can we try with the `/addresses` feature please?

---

Fixed in e14363f2e5b5178d59c1c4cc55f3c6dc1d79732f.